test/PINO_ode_tests.jl

```diff
@@ -0,0 +1,265 @@
+using Test
+using OptimizationOptimisers
+using Lux
+using Statistics, Random
+using LuxNeuralOperators
+using NeuralPDE
+
+# dG(u(t, p), t) = f(G,u(t, p))
+@testset "Example du = cos(p * t)" begin
+    equation = (u, p, t) -> cos(p * t)
+    tspan = (0.0f0, 1.0f0)
```

Comment:
Why? In others tests PINNS ODE and many others we use Float32. And more over we need initialize neural network with weights Float64 then it will make sense. Otherwise, we multiply float64 by float32 in the neural network and do not get increase accuracy, only an unnecessary conversion. This is not critical for one-dimensional ODE problems, but for multi-dimensional ones with PDE, training in float64 will take much longer and it can be too long for test time. So I don’t really understand why use Float64 everywhere, if only one test and then it’s not necessary

Reply:
Alright, I would be fine with tests but not with tutorials as people usually copy paste.

Reply:
Oh, I didn't see some comments from last preview. I will fix it in soon.